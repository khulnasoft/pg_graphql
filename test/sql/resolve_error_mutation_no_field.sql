begin;

    select graphql.resolve($$
    mutation {
       insertDNE(object: {
        email: "o@r.com"
      }) {
        id
      }
    }
    $$);

rollback;
