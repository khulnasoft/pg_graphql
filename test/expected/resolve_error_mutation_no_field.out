begin;
    select graphql.resolve($$
    mutation {
       insertDNE(object: {
        email: "o@r.com"
      }) {
        id
      }
    }
    $$);
                             resolve                              
------------------------------------------------------------------
 {"data": null, "errors": [{"message": "Unknown type Mutation"}]}
(1 row)

rollback;
